src/components/common/ColonyHome/ColonyFundingWidget/ColonyFundingWidget.tsx

```diff
@@ -35,11 +35,15 @@ const ColonyFundingWidget = (/* { currentDomainId }: Props */) => {
 
   const {
     name,
-    tokens,
+    tokens: colonyTokenItems,
```

Comment:
@rdig I'm not a fan of how Amplify resolves many-to-many relations. You'd expect the `tokens` field to contain `Token` object, but it's actual return type is `ModelColonyTokensConnection`. Maybe we should rename it at the query level?